prboom2/cmake/FindOgg.cmake

```diff
@@ -48,32 +50,60 @@ pkg_check_modules(PC_OGG QUIET ogg)
 find_path(
   Ogg_INCLUDE_DIR
   NAMES ogg/ogg.h
-  HINTS ${PC_OGG_INCLUDEDIR})
+  HINTS "${PC_OGG_INCLUDEDIR}"
+)
+
+find_file(
+  Ogg_DLL
+  NAMES ogg.dll libogg.dll libogg-0.dll
```

Comment:
What are the consequences if there is a version with a different name for one of these find commands? Does any part of the build / install process fail?